feature-wallet-api/src/main/java/io/novafoundation/nova/feature_wallet_api/domain/model/CoinRate.kt

```diff
@@ -0,0 +1,15 @@
+package io.novafoundation.nova.feature_wallet_api.domain.model
+
+import io.novafoundation.nova.runtime.multiNetwork.chain.model.Chain
+import java.math.BigDecimal
+import java.math.BigInteger
+
+open class CoinRate(val rate: BigDecimal)
+
+class CoinRateChange(val recentRateChange: BigDecimal, value: BigDecimal) : CoinRate(value)
+
+class HistoricalCoinRate(val timestamp: Long, value: BigDecimal) : CoinRate(value)
```

Comment:
Why do we need such hierachy of classes? To me, `CoinRate` is not a parent for `CoinRateChange`,it is more like 
`class CoinRateChange(val recentRateChange: BigDecimal, val rate: CoinRate)`. Similar thing with `HistoricalCoinRate`